internal/controllers/controllers_default.go

```diff
@@ -44,3 +45,7 @@ func NewWarehousesController(service models.WarehouseService) *warehouses_contro
 func NewEmployeesController(sv models.EmployeesService) *employees_controller.EmployeesController {
 	return employees_controller.NewEmployeesDefault(sv)
 }
+
+func NewPurchaseOrdersController(service models.PurchaseOrdersService) *purchase_orders_controller.PurchaseOrdersController {
+	return &purchase_orders_controller.PurchaseOrdersController{Service: service}
+}
```

Comment:
que tal usar a função de inicializar o controller ao invés disso?
```suggestion
func NewPurchaseOrdersController(sv models.PurchaseOrdersService) *purchase_orders_controller.PurchaseOrdersController {
	return purchase_orders_controller.NewPurchaseOrdersController(sv)
}
```